api-catalog-ui/frontend/src/reducers/loading-reducer.jsx

```diff
@@ -10,7 +10,8 @@
 
 const loadingReducer = (state = {}, action = {}) => {
     const { type } = action;
-    const matches = /(.*)_(REQUEST|SUCCESS|FAILURE|FAILED)/.exec(type);
+
+    const matches = /(.*)_(REQUEST|SUCCESS|FAILURE|FAILED|INVALIDPASSWORD)/.exec(type);
```

Comment:
Not totally sure, but shouldn't you add also match for `USERS_LOGIN_INIT` and `USERS_LOGIN_VALIDATE`?

Reply:
I don't think so, they are not being used in the context of HTTP requests so they should not trigger loading spinner.  The spinner was working properly while I was testing it. However, it could be that I don't understand this concept very well.

Reply:
I think you're right, it's in the context of an API call according to [this](https://medium.com/stashaway-engineering/react-redux-tips-better-way-to-handle-loading-flags-in-your-reducers-afda42a804c6), so I think we're good.